debug/Dockerfile

```diff
@@ -16,11 +16,21 @@ RUN apk add --no-cache \
 RUN mkdir -p -m 0600 ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
 
 # Install https://github.com/ellistarn/home
-RUN --mount=type=ssh sh -c \
-    "$(wget https://raw.githubusercontent.com/ellistarn/home/main/bin/install-home.sh -O -)"
+#RUN --mount=type=ssh sh -c \
```

Comment:
Leave this guy in here.

Reply:
my bad, broke for me, but i'll keep it in there.